components/combobox/stories/combobox.stories.js

```diff
@@ -127,7 +141,34 @@ export default {
 		isLoading: false,
 		isDisabled: false,
 		showFieldLabel: false,
-		fieldLabelText: "Select location"
+		fieldLabelText: "Select location",
+		content: [
+			Menu({
+				role: "listbox",
+				subrole: "option",
+				isSelectable: true,
+				items: [
```

Comment:
We should also pass `size` to menu so it changes when we toggle the control for combobox